init.lua

```diff
@@ -524,32 +552,27 @@ end
 
 local function progressive_show_recipe(recipe, inv_items)
 	local inv_items_size = #inv_items
-	for _, item in pairs(recipe.items) do
+	for _, item_string in pairs(recipe.items) do
```

Comment:
Keep `item`.

Reply:
same as above. The value is not cleaned. The value is not checked if it is registered item, registered alias, unknown item or group string